src/server/contest/views.py

```diff
@@ -159,12 +160,12 @@ def upload(request):
             '_' + str(problem_.problem_id) + '.c'
 
         uploaded_filedata = request.FILES['submission_file']
-        submissions_fold = Path('contest/submissions')
-        # creates /contest/submissions folder if does not exist
+        submissions_fold = Path('contest/submissions/latest')
```

Comment:
Confusing filename, should be `latest_submissions_fold` instead of `submissions_fold`.